app/services/hyrax/custom_queries/find_file_metadata.rb

```diff
@@ -1,13 +1,15 @@
 module Hyrax
   module CustomQueries
     class FindFileMetadata
-      # Use:
+      # @example
```

Comment:
This should move above `class FindFileMetadata` to act as documentation for the class.

Reply:
DONE - moved to top of file